File: app/src/main/java/kim/hsl/component/MainActivity.java
package kim.hsl.component;

import android.app.Activity;
import android.os.Bundle;

import androidx.annotation.Nullable;

import kim.hsl.route_core.Router;
import kim.hsl.router_annotation.Route;

@Route(path = "/app/MainActivity")
public class MainActivity extends Activity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // 初始化路由表
        Router.init(getApplication());
    }
}

File: library1/src/main/java/kim/hsl/library1/MainActivity.kt
package kim.hsl.library1

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kim.hsl.router_annotation.Route

@Route(path = "/library1/MainActivity")
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}